src/context/GraphQueryHandler.js

```diff
@@ -28,7 +28,7 @@ export class GraphQueryHandler extends PureComponent {
   }
 
   getYearsRange = (startDate: string = null) => {
```

Comment:
y si le pasas por default `'2018-01-01'` a `startDate`? 
algo como:
```suggestion
  getYearsRange = (startDate: string = '2018-01-01') => {
    const startingYear = new Date(startDate).getUTCFullYear();
```